openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-tlmviewer/src/tools/TlmViewer/TlmViewer.vue

```diff
@@ -196,10 +196,6 @@ export default {
       dragHandle: '.v-system-bar',
     })
     this.grid.on('dragEnd', this.refreshLayout)
-    const previousConfig = localStorage['lastconfig__telemetry_viewer']
-    if (previousConfig) {
-      this.openConfiguration(previousConfig)
-    }
```

Comment:
Why are you removing this?

Reply:
This was really bugging me.  I didn't want the the same config to keep coming up and there is no built in method to stop it.   Needs to be redone with some kind of pin on/off.